content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -9,176 +9,268 @@ With a down payment, the buyer pays a portion of the total amount owed while agr
+
 .. note::
-   Invoices are automatically created in drafts, so they can be reviewed before validation.
+   Invoices are automatically created as drafts, so they can be reviewed before validation.
 
 On the :guilabel:`Create invoices` pop-up, there are 3 options to choose from in the
 :guilabel:`Create Invoice` field:
 
 - :guilabel:`Regular invoice`
-   .. note::
-      If :guilabel:`Regular Invoice` is selected the other fields disappear, as they only pertain to
-      down payment configurations.
 - :guilabel:`Down payment (percentage)`
 - :guilabel:`Down payment (fixed amount)`
 
-In terms of a down payment, it can either be a fixed amount or a percentage of the total amount.
+.. note::
+   If :guilabel:`Regular Invoice` is selected the other fields disappear, as they only pertain to
+   down payment configurations.
+
+   .. image:: down_payment/create-invoices-popup-form-regular-option.png
+      :align: center
+      :alt: The Regular Invoice option selected on the Create invoices pop-up form.
 
-Once the desired down payment option is selected in the :guilabel:`Create Invoice` field, designate
-the desired amount (either as a percentage or fixed amount) in the :guilabel:`Down Payment Amount`
-field.
+Initial down payment request
+============================
 
-Then, select the appropriate income account for the invoice in the :guilabel:`Income Account` field,
-and add a tax amount, if necessary, in the :guilabel:`Customer Taxes` field. These settings will be
-reused for future down payments.
+On the :guilabel:`Create invoices` pop-up form, the down payment options are:
 
-.. image:: down_payment/create-invoices-popup-form.png
+- :guilabel:`Down payment (percentage)`
+- :guilabel:`Down payment (fixed amount)`
+
+Once the desired down payment option is selected in the :guilabel:`Create Invoice` field on the
+pop-up form, designate the desired amount (either as a percentage or fixed amount) in the
+:guilabel:`Down Payment Amount` field.
+
+Then, select the appropriate income account for the invoice in the :guilabel:`Income Account` field.
+Then, select a tax amount, if necessary, in the :guilabel:`Customer Taxes` drop-down field.
+
+.. image:: down_payment/create-invoices-popup-form-filled-out.png
    :align: center
-   :alt: How to configure a down payment on Odoo Sales.
+   :alt: A create invoices pop-up form with down payment fields filled in with information.
 
-.. important::
-   When a down payment is first requested, a new product called :guilabel:`Down payment` is created.
-   This product is registered as a :guilabel:`Service` product, with an invoicing policy of
-   :guilabel:`Ordered quantities`. This product can be edited and modified at any time.
+When all fields are filled in with the desired information, click the :guilabel:`Create Draft
+Invoice` button. Upon clicking :guilabel:`Create Draft Invoice` on the :guilabel:`Create invoices`
+pop-up form, Odoo reveals the :guilabel:`Customer Invoice Draft`.
```

Comment:
```suggestion
Once all fields are filled in with the desired information, click the :guilabel:`Create Draft
Invoice` button. Upon clicking this button, Odoo reveals the :guilabel:`Customer Invoice Draft`.
```
This read as a little long-winded for me, so this is just a suggestion to shorten it a bit